include/tsid/bindings/python/contacts/expose-contact.hpp

```diff
@@ -21,19 +21,22 @@
 #include "tsid/bindings/python/contacts/contact-6d.hpp"
 #include "tsid/bindings/python/contacts/contact-point.hpp"
 #include "tsid/bindings/python/contacts/contact-two-frame-positions.hpp"
+#include "tsid/bindings/python/contacts/measured-6Dwrench.hpp"
 
 namespace tsid {
 namespace python {
 void exposeContact6d();
 void exposeContactPoint();
 void exposeContactTwoFramePositions();
+void exposeMeasured6Dwrench();
 
 inline void exposeContact() {
   exposeContact6d();
   exposeContactPoint();
   exposeContactTwoFramePositions();
+  exposeMeasured6Dwrench();
```

Comment:
For consistency with the already existing "Contact6d" I would prefer using "Measured6dWrench", without capitalizing the "d" after 6.